src/data/colleges/il.ts

```diff
@@ -42,7 +42,7 @@ const FLcourses: readonly string[] = [
 
 const ilrRequirements: readonly CollegeOrMajorRequirement[] = [
   {
-    name: 'First Year Core Requirements',
+    name: 'First Year Core Fart',
```

Comment:
I think we'll have to discuss this change with the marketing team

Reply:
😂

Reply:
I was testing to see if my code changed the text on the website. I didn't know I pushed this...
Although, I should have reverted back this change on a later commit
https://github.com/cornell-dti/course-plan/pull/793/commits/bf8dfd1a2253fa0fa821fd03e5df983cf4a3c964